renderer/modules/settings/installations/components/Installations/Installations.view.tsx

```diff
@@ -34,92 +36,101 @@ export default function InstallationsView(
-        </TableContainer>
-      </Card>
+			<Card>
+				<div className={classes.header}>
+					<form className={classes.form} noValidate autoComplete="off">
+						<TextField
+							id="outlined-basic"
+							label="Search versions"
+							variant="outlined"
+							value={props.query}
+							onChange={props.queryHandler}
+							className={classes.textField}
+						/>
+					</form>
+					<Typography variant="body2" component="p">
+						INSTALLED TOOLS
+					</Typography>
+					<Typography gutterBottom variant="h5" component="h2">
+						Devon IDE Versions
+					</Typography>
+				</div>
+				<TableContainer component={Paper}>
+					<Table>
+						<TableHead>
+							<TableRow>
+								<TableCell>VERSION NAMES</TableCell>
+								<TableCell></TableCell>
+								<TableCell>RELEASE DATE</TableCell>
+								<TableCell>RELEASE NOTES</TableCell>
+								<TableCell align="center">UPDATE</TableCell>
+								<TableCell align="center">DOWNLOAD</TableCell>
+							</TableRow>
+						</TableHead>
+						<TableBody>
+							{(props.rowsPerPage > 0
+								? props.installations.slice(
+										props.page * props.rowsPerPage,
+										props.page * props.rowsPerPage + props.rowsPerPage
+									)
+								: props.installations
+							).map((installation: DevonIdeScripts, index: number) => (
+								<TableRow key={index}>
+									<TableCell>{installation.version}</TableCell>
+									<TableCell>
+										{installation.path ? (
+											<IconButton
+												onClick={() => {
+													props.viewIdeHandler(installation.path);
+												}}
+											>
+												<VisibilityIcon fontSize="small" />
+											</IconButton>
+										) : null}
+									</TableCell>
+									<TableCell>{installation.updated}</TableCell>
+									<TableCell>
+										{installation.changelog ? (
+											<Changelog version={installation.version} />
+										) : null}
+									</TableCell>
+									<TableCell align="center">
+										<AcceptButton>UPDATE</AcceptButton>
+									</TableCell>
+									<TableCell align="center">
+										{installation.url ? (
+											<AcceptButton
+												startIcon={<GetApp />}
+												href={installation.url}
+											>
+												Download
+											</AcceptButton>
+										) : (
+											<AcceptButton
+												onClick={() => props.uninstallHandler(installation.path)}
+											>
+												Uninstall
+											</AcceptButton>
+										)}
+									</TableCell>
+								</TableRow>
+							))}
+						</TableBody>
+						<TableFooter>
+							<TableRow>
+								<TablePagination
+									rowsPerPageOptions={[5, 10, 25]}
+									count={props.installations.length}
+									rowsPerPage={props.rowsPerPage}
+									page={props.page}
+									onChangePage={props.pageChangehandler}
+									onChangeRowsPerPage={props.rowsPerPageChangeHandler}
+								/>
+							</TableRow>
+						</TableFooter>
+					</Table>
+				</TableContainer>
+			</Card>
```

Comment:
It seems that something happened here with the formatting

![image](https://user-images.githubusercontent.com/43032659/92376168-77c74c80-f102-11ea-92fb-376247e3afeb.png)